js/menus/menuTemplate.js

```diff
@@ -333,6 +334,22 @@ function getTemplate(app) {
         }
     });
 
+    // Window/View menu -> separator
+    template[index].submenu.push({
+        type: 'separator',
+    });
+
+    // Window - View menu -> memoryRefresh
+    template[index].submenu.push({
+        label: 'Memory Refresh',
```

Comment:
I think a better copy would be something like `Optimize Performance` or `Lower Memory Footprint` or something like that. `Memory Refresh` literally makes no sense.

I would recommend talking to product about this.

Reply:
@KiranNiranjan Please see this (https://perzoinc.atlassian.net/browse/ELECTRON-456) approved by Nancy.